app/views/facilities/show.html.haml

```diff
@@ -2,15 +2,17 @@
   = current_facility
 
 - if acting_as? || session_user.try(:operator_of?, current_facility)
-  - instruments = current_facility.instruments.active_plus_hidden
-  - items       = current_facility.items.active_plus_hidden
-  - services    = current_facility.services.active_plus_hidden
-  - bundles     = current_facility.bundles.active_plus_hidden
+  - instruments    = current_facility.instruments.active_plus_hidden
+  - items          = current_facility.items.active_plus_hidden
+  - services       = current_facility.services.active_plus_hidden
+  - timed_services = current_facility.timed_services.active_plus_hidden
+  - bundles        = current_facility.bundles.active_plus_hidden
```

Comment:
I feel like it's possible that facility managers would find having the old bundles visible to be useful--I don't know if they use that for anything currently but wouldn't want to break it if they do.